content/index.html.haml

```diff
@@ -36,11 +36,11 @@ homepage: true
 = partial('downloadbanner.html.haml')
 
 = partial('projectjumbotron.html.haml',
-  :href => '/blog/2017/12/31/new-year/',
-  :title => 'Welcome to 2018!',
-  :intro => "Jenkins project congratulates all users and contributors with the New Year. In 2017 we had 60 weekly and 13 LTS releases with 305 fixes/enhancements only in the core. There were 2605 plugin releases, and 215 <b>NEW</b> plugins were added to the Update Center.<p/> This year we released major features like BlueOcean and Declarative Pipeline. We also kicked off new initiatives like Jenkins Enhancement Proposal. Stay tuned, there is much more to come next year!",
-  :image => '/images/post-images/2017-12-31-new-year/ny_tree.png',
-  :call_to_action => {:text => 'More Info', :href => '/blog/2017/12/31/new-year'})
+  :href => '/blog/2018/01/13/jep-200/',
+  :title => 'Notice for Jenkins 2.102+',
+    :intro => "JEP-200: 'Switch Remoting/XStream blacklist to a whitelist' has been integrated into Jenkins 2.102. If you use weekly releases, be careful when updating to this version or newer ones. There is a <b>HIGH RISK</b> of regressions in plugins.<p/>See the linked blogpost for upgrade guidelines and the list of affected plugins. Jenkins team is working on stabilizing the plugin ecosystem.",
+  :image => '/images/blueocean/blueocean-successful-pipeline.png',
```

Comment:
I have no good images for that, so I just reverted the image to the previous one